### PR TITLE
Support importing from a directory w/o imports. Fixes #245.

### LINKED ISSUE #245: Import all graphql files from a directory
I'm looking at `graphqlgen` that defers to `importSchema` to load its schemas.

Right now we just have `*.graphql` files in a `data/schemas` directory without any of the `# import * from ...` comments.

What I'd like to do is just `importSchema('data/schemas')` and have it realize its a directory and just pull in all the `.graphql` files recursively and combine them together.

Does that seem reasonable?

### PR DESCRIPTION
**Pull request by stephenh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,33 +6,40 @@
   },
   "license": "MIT",
   "repository": "git@github.com:graphcool/graphql-import.git",
-  "files": ["dist"],
+  "files": [
```
Comment (stephenh) [inline package.json]: Er, didn't meant to commit these changes.

**Comment by scroobius-pip** [comment]
This pull request has been hanging since last year. Any reason why ?. This feature seems pretty usefull.

**Comment by nfantone** [comment]
I believe this should be superseded by, and thus closed in favour of, #192.

**Comment by ardatan** [comment]
This can be done by using glob patterns more efficiently! Thank you for your contribution :)

**Pull request closed without merging**